docs/Mining.md

```diff
@@ -142,13 +142,16 @@ cd base_miner/UCF/ && python train_detector.py
 ```
 The model with the lowest validation accuracy will be saved to `base_miner/UCF/logs/training/<experiment_name>/`.<br>
 
-In this directory, you will find your model weights (`ckpt_best.pth`) and training configuration (`config.yaml`).
+In this directory, you will find your model weights (`ckpt_best.pth`) and training configuration (`config.yaml`). Note that
+the training config, e.g. `config.yaml`, is different from the detector config, e.g. `ucf.yaml`.
+
+You will need to put the path to the training configuration file `config.yaml` into the detector configuration file `ucf.yaml` to instantiate `UCFDetector(DeepfakeDetector`, as it uses the settings from training time to reconstruct the correct model layer shapes for inference.
```

Comment:
[Suggestion] move this to a UCF specific note in the `Deploy Your Model` section, or add something prefacing this saying this is something that must be done after training/before deploying